OpenRA.Mods.Common/UpdateRules/Rules/20201213/ConvertBoundsToWDist.cs

```diff
@@ -0,0 +1,55 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2021 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace OpenRA.Mods.Common.UpdateRules.Rules
+{
+	public class ConvertBoundsToWDist : UpdateRule
+	{
+		public override string Name => "Convert Interactable and Selection bounds from pixels to WDist.";
+
+		public override string Description =>
+			"The Bounds and DecorationBounds fields on Interactable and Selectable have been converted from pixels to WDist.\n" +
+			"All bounds must be scaled by 1024 divided by your mod tile size.";
```

Comment:
Not entirely true according to `var tileScale = grid.Type == MapGridType.RectangularIsometric ? 1448 : 1024;`?